modules/nw-egressnetworkpolicy-create.adoc

```diff
@@ -25,25 +37,34 @@ If the project already has an EgressNetworkPolicy object defined, you must edit
 policy rules.
 .. In the file you created, define an egress policy object.
 
-. Enter the following command to create the policy object:
+. Enter the following command to create the policy object. Replace `<policy-name>` with the name of the policy and `<project>` with the project that the rule applies to.
```

Comment:
Throughout, the replaceables should [use underscores](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output).